{
  logger.error("RetryOperationCallback call was interrupted for account {}: {}",paymentStateContext.getAccount().getExternalKey(),e.getMessage());
  return new OperationException(e,getOperationResultOnException(paymentStateContext));
}
